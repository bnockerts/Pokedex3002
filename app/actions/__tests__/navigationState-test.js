import navigationStateActions from '../navigationState';

it('creates a PUSH_ROUTE action, string', () => {
  const testRouteName = 'testRoute';

  expect(navigationStateActions.navigatePush(testRouteName))
    .toMatchSnapshot();
});

it('creates a PUSH_ROUTE action, object', () => {
  const testRoute = {
    key: 'testRoute',
    title: 'testTitle',
    other: 'data'
  };

  expect(navigationStateActions.navigatePush(testRoute))
    .toMatchSnapshot();
});

it('creates a POP_ROUTE action', () => {
  expect(navigationStateActions.navigatePop())
    .toMatchSnapshot();
});
