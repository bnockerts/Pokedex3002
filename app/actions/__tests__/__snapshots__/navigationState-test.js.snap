exports[`test creates a POP_ROUTE action 1`] = `
Object {
  "type": "POP_ROUTE",
}
`;

exports[`test creates a PUSH_ROUTE action, object 1`] = `
Object {
  "state": Object {
    "key": "testRoute",
    "other": "data",
    "title": "testTitle",
  },
  "type": "PUSH_ROUTE",
}
`;

exports[`test creates a PUSH_ROUTE action, string 1`] = `
Object {
  "state": Object {
    "key": "testRoute",
    "title": "testRoute",
  },
  "type": "PUSH_ROUTE",
}
`;
